docs/source/using/interacting-with-openshift.adoc

```diff
@@ -201,37 +197,14 @@ Make sure to replace `IP-ADDRESS` in the above example with the IP address of yo
 You can retrieve the IP address by running the link:../command-ref/minishift_ip{outfilesuffix}[`ip`] command.
 
 [[persistent-volumes]]
-== Persistent volumes
-
-Minishift supports https://docs.openshift.org/latest/dev_guide/persistent_volumes.html[PersistentVolumes]
-of type `hostPath`. These persisten volumes are mapped to a directory inside the Minishift VM.
-
-The MiniShift VM boots into a tmpfs, so most directories will not be persisted across
-reboots, for example if you use `minishift stop`. However, MiniShift is configured to
-persist OpenShift-specific configuration files and some Docker files.
-
-The files that are persisted are stored in the following host directories:
+== Persistent Volumes
 
-- `/var/lib/minishift`
-- `/var/lib/docker`
-
-The following example shows persistent volume configuration to persist data
-in the `/var/lib/minishift` directory:
-
-[source, yaml]
-----
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: pv
-spec:
-  accessModes:
-    - ReadWriteOnce
-  capacity:
-    storage: 5Gi
-  hostPath:
-    path: /var/lib/minishift/pv
-----
+As part of the <<interacting-with-openshift#openshift-client-binary,cluster up>> provisioning
```

Comment:
-1 Let's remove the whole section, no need to duplicate things. The old persistence config section is outdated anyways.